### ISSUE TITLE
package.json entry crashes npm install

### ISSUE THREAD

**Issue by jQrgen** [WebsiteBeaver/CordovaCall#14]
Hello

The plugin puts two lined in package.json:
```js
{
    "dependencies": {
        ......
        "com.dmarc.cordovacall": "~1.0.4"
    },
    .......
    "cordova": {
        "com.dmarc.cordovacall": {}
    }
}
```

The dependencies injection causes npm install to fail, because the package com.dmarc.cordovacall is [non-existant](https://www.npmjs.com/search?q=com.dmarc.cordovacall) in the npm registry.

```js
$ npm install
npm ERR! code E404
npm ERR! 404 Not Found: com.dmarc.cordovacall@~1.0.4
```

Thanks for any help with this.

**Comment by D-Marc1** [comment]
Hey, why don't you just do `cordova plugin add cordova-call`?

**Comment by jQrgen** [comment]
Thanks

**Comment by HugoHeneault** [comment]
So I did that but the plugin is saved in the config.xml as `com.dmarc.cordovacall` so when I do a `cordova platform add android` on another computer I got the error

```
Discovered plugin "com.dmarc.cordovacall" in config.xml. Adding it to the project
Failed to restore plugin "com.dmarc.cordovacall" from config.xml. You might need to try adding it again. Error: Failed to fetch plugin com.dmarc.cordovacall@~1.0.8 via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 1 Error output:
npm ERR! code E404
npm ERR! 404 Not Found: com.dmarc.cordovacall@~1.0.8
```

What's wrong? :(

**Comment by D-Marc1** [comment]
Remove and then re-add CordovaCall to your project

**Comment by HugoHeneault** [comment]
Done it. Multiple times. 
cordova-call is always saved in `config.xml` as `com.dmarc.cordovacall`. So next `platform add` will break it again...

**Comment by dmarcs** [comment]
I just fixed this issue in 1.1.0. It now saves the plugin as `cordova-call` instead of `com.dmarc.cordovacall`. Now you can remove and reinstall the plugin with `cordova plugin add cordova-call`, and there shouldn't be any issues.

**Comment by HugoHeneault** [comment]
That's great ! I'll try it on Monday. Thanks!!
